src/app/store/slices/storage/storage.thunks/uploadItemsThunk.ts

```diff
@@ -112,12 +112,13 @@ const prepareFilesToUpload = async ({
   workspaceToken?: string;
 }): Promise<{ filesToUpload: FileToUpload[]; zeroLengthFilesNumber: number }> => {
   const filesToUpload: FileToUpload[] = [];
-  const storageClient = SdkFactory.getNewApiInstance().createNewStorageClient();
-
   let parentFolderContent;
 
   if (!disableDuplicatedNamesCheck) {
-    const [parentFolderContentPromise] = storageClient.getFolderContentByUuid(parentFolderId, false, workspaceToken);
+    const [parentFolderContentPromise] = newStorageService.getFolderContentByUuid({
+      folderUuid: parentFolderId,
+      workspacesToken: workspaceToken,
+    });
```

Comment:
Same here, remove this change, as in the PR https://github.com/internxt/drive-web/pull/1282 is not used anymore